### ISSUE TITLE
How do we use this?

### ISSUE THREAD

**Issue by MartinDawson** [relayjs/eslint-plugin-relay#24]
Nothing explaining in the Readme.md about how to get this to work or any examples.

**Comment by pranaygp** [comment]
@MartinDawson you should be able to simply `yarn add eslint-plugin-relay` and then add the plugin to your [elsint configuration](https://eslint.org/docs/user-guide/configuring#configuring-plugins).

You can also turn on/off individual rules contained within this plugin, which is also documented over at [eslint](https://eslint.org/docs/user-guide/configuring#configuring-rules)

**Comment by kassens** [comment]
If someone's interested in a quick PR to add instructions/links to the doc, I'm happy to accept them!

**Comment by MartinDawson** [comment]
@pranaygp i'v done that and added the config in my `.eslint.js` file and no validation still.

**Comment by kassens** [comment]
@MartinDawson: can you try adding the rule:
```
  "relay/graphql-syntax": "error"
```
and then have a GraphQL syntax error somewhere in a linted file, such as

```
const query = graphql`Boom`;
```

which should trigger this rule.

**Comment by MartinDawson** [comment]
@kassens Thanks. This worked. Added a PR for it.
